pkg/operator/controllers/suite_test.go

```diff
@@ -99,11 +99,13 @@ var _ = BeforeSuite(func() {
 	Expect(err).ToNot(HaveOccurred())
 
 	err = (&GameServerReconciler{
-		Client:                 k8sManager.GetClient(),
-		Scheme:                 k8sManager.GetScheme(),
-		PortRegistry:           portRegistry,
-		Recorder:               k8sManager.GetEventRecorderFor("GameServerReconciler"),
-		GetNodeDetailsProvider: func(_ context.Context, _ client.Reader, _ string) (string, int, error) { return "testPublicIP", 0, nil },
+		Client:       k8sManager.GetClient(),
+		Scheme:       k8sManager.GetScheme(),
+		PortRegistry: portRegistry,
+		Recorder:     k8sManager.GetEventRecorderFor("GameServerReconciler"),
+		GetNodeDetailsProvider: func(_ context.Context, _ client.Reader, _ string) (string, string, int, error) {
+			return "testNodeName", "testPublicIP", 0, nil
```

Comment:
(We should have done this already but anyway)
Can we modify any of the existing tests to check for the "testNodeName" and "testPublicIP" strings? I think it was omitted since we were returning strongly typed string, int, err but now that we're returning two strings, it might make sense.